### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Build Stage
```
Comment (xansec) [inline Dockerfile:1]: Please do not use git clone or `ldd`.

File: mayhem/json2yaml.mayhemfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+image: ghcr.io/capuanob/yaml-fuzz:latest
```
Comment (xansec) [inline mayhem/json2yaml.mayhemfile:1]: Please omit image field. Tests -> testsuite.

File: mayhem/yaml2json.mayhemfile [hunk 3]
```diff
@@ -0,0 +1,8 @@
+image: ghcr.io/capuanob/yaml-fuzz:latest
```
Comment (xansec) [inline mayhem/yaml2json.mayhemfile:1]: Please omit image field. Tests -> testsuite.

**Review by xansec — CHANGES_REQUESTED** [review]
You have a decent sized corpus, but the files are small. I'll let you decide if you want to trim this or not. Please make the requested changes.

**Comment by capuanob** [comment]
@xansec Done!

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**